src/app/containers/StoryPromo/index.jsx

```diff
@@ -6,11 +6,28 @@ import StoryPromoComponent, {
   Link,
 } from '@bbc/psammead-story-promo';
 import { storyItem } from '../../models/propTypes/storyItem';
-import StoryPromoFigure from './Figure';
+import FigureContainer from '../Figure';
 import Timestamp from '../Timestamp';
 import { ServiceContext } from '../../contexts/ServiceContext';
 import deepGet from '../../helpers/json/deepGet';
 
+const renderImage = imageValues => {
+  if (!imageValues) {
+    return null;
+  }
+  const ratio = (imageValues.height / imageValues.width) * 100;
+  const src = `https://ichef.bbci.co.uk/news/660${imageValues.path}`;
+
+  return (
+    <FigureContainer
+      alt={imageValues.altText}
+      ratio={ratio}
+      src={src}
+      {...imageValues}
```

Comment:
Should we be enabling lazy loading in the figure container?

Reply:
Is lazy loading a requirement on `StoryPromo`?

Reply:
Not been mentioned yet really, Check with product but I think we should :)

Reply:
Maybe we can lazyload everything except the first story?